src/cli/mod.rs

```diff
@@ -108,6 +108,12 @@ pub(crate) struct FlakeHubPushCli {
 
     #[clap(long, env = "FLAKEHUB_PUSH_INCLUDE_OUTPUT_PATHS", value_parser = EmptyBoolParser, default_value_t = false)]
     pub(crate) include_output_paths: bool,
+
+    // Gitlab has a concept of subgroups, which enables repo names like https://gitlab.com/a/b/c/d/e/f/g. By default,
+    // flakehub-push would parse that to flake name `a/b-c-d-e-f-g`. This flag/environment variable provides a
+    // mechanism to disable this behavior.
+    #[clap(long, env = "FLAKEHUB_PUSH_DISABLE_SUBGROUPS", default_value_t = false)]
```

Comment:
```suggestion
    #[clap(long, env = "FLAKEHUB_PUSH_RENAME_SUBGROUPS", default_value_t = false)]
```

how about something like this? We're not really disabling them, but we are disabling the "fix the name" part.

Reply:
I've opted for `FLAKEHUB_PUSH_DISABLE_RENAME_SUBGROUPS`